lnspec_py/basic_type/int.py

```diff
@@ -16,16 +16,24 @@ def decode(self):
 
 
 class u16Int(Integer):
-    def __init__(self, val):
+    def __init__(self, val: int) -> None:
         self.val = val
 
     def encode(self):
         self.val = int.to_bytes(self.val, 2, "big")
 
-    def decode(self):
+    def decode(self) -> "u16Int":
         if type(self.val) == str:
             self.val = bytes.fromhex(self.val)
         self.val = int.from_bytes(self.val, "big")
+        return self
+
+    @staticmethod
+    def decode_with_HexStr(hex_str: str) -> "u16Int":
```

Comment:
```suggestion
    def decode_with_hex_str(hex_str: str) -> Tuple["u16Int", str]:
```

Check the correct syntax of the Tuple :) just on flight review